bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1354,18 +1371,31 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 </finidat>
 
 <!-- CTSM6_0 - use clm6_0_GSWP3v1 BgcCrop at f09 for all clm5_1/clm6_0 options -->
-<!-- use_init_interp true because the file was created with ctsm5.3 datasets -->
 <finidat hgrid="0.9x1.25" mask="gx1v7" use_cn=".true." use_cndv=".false." use_fates=".false."
          ic_ymd="18500101" sim_year="1850" do_transient_pfts=".false." use_excess_ice=".true."
          ic_tod="0" glc_nec="10" use_crop=".true." irrigate=".false."
-         phys="clm5_1" use_init_interp=".true."
->lnd/clm2/initdata_esmf/ctsm5.3/ctsm52026_f09_pSASU.clm2.r.0421-01-01-00000.nc
+         phys="clm5_1"
+>lnd/clm2/initdata_esmf/ctsm5.3/ctsm53n04ctsm52028_f09_pSASU.clm2.r.0161-01-01-00000.nc
 </finidat>
 <finidat hgrid="0.9x1.25" mask="gx1v7" use_cn=".true." use_cndv=".false." use_fates=".false."
          ic_ymd="18500101" sim_year="1850" do_transient_pfts=".false." use_excess_ice=".true."
          ic_tod="0" glc_nec="10" use_crop=".true." irrigate=".false."
-         phys="clm6_0" use_init_interp=".true."
->lnd/clm2/initdata_esmf/ctsm5.3/ctsm52026_f09_pSASU.clm2.r.0421-01-01-00000.nc
+         phys="clm6_0"
+>lnd/clm2/initdata_esmf/ctsm5.3/ctsm53n04ctsm52028_f09_pSASU.clm2.r.0161-01-01-00000.nc
+</finidat>
+<!-- Corresponding ne30 -->
+<finidat hgrid="ne30np4.pg3" mask="t232" use_cn=".true." use_cndv=".false." use_fates=".false."
+         sim_year="1850" do_transient_pfts=".false." use_excess_ice=".true."
+         use_crop=".true." irrigate=".false."
+         phys="clm6_0"
+>lnd/clm2/initdata_esmf/ctsm5.3/ctsm53n04ctsm52028_ne30pg3t232_pSASU.clm2.r.0121-01-01-00000.nc
+</finidat>
+<!-- Corresponding f19 -->
+<finidat hgrid="1.9x2.5" mask="gx1v7" use_cn=".true." use_cndv=".false." use_fates=".false."
+         sim_year="1850" do_transient_pfts=".false." use_excess_ice=".true."
+         use_crop=".false." irrigate=".false."
+         phys="clm6_0"
+>lnd/clm2/initdata_esmf/ctsm5.3/ctsm530_f19_PPE_pSASU.clm2.r.0161-01-01-00000.nc
```

Comment:
Removed use_init_interp from these but it shows up as true in f19 and ne30.

All three 1850 manual tests passed.

Reply:
For f19 need to change the test to Bgc (no crop).

Reply:
Finish this test.

Rename all finidats to include mask, Bgc/Sp/BgcCrop, exice, clm60, and remove -00000 e.g.
`ctsm530_f19_PPE_pSASU.clm2.r.0161-01-01-00000.nc` -->
`ctsm530_f19_g17_Bgc_exice_pSASU.clm60.r.0161-01-01.nc`

Reply:
ne30: replace mask everywhere in namelist_defaults_ctsm.xml with its long name.